custom_components/easee/diagnostics.py

```diff
@@ -9,11 +9,27 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.device_registry import DeviceEntry
 
+from .const import DOMAIN
+
 TO_REDACT = {
     CONF_PASSWORD,
     CONF_USERNAME,
 }
 TO_REDACT_DATA = {}
+TO_REDACT_SITES = {
+    "id",
+    "siteKey",
+    "installerName",
+    "installerPhoneNumber",
+    "ownerName",
+    "ownerPhoneNumber",
+    "siteId",
+    "masterBackPlateId",
+    "street",
+    "buildingNumber",
+    "zip",
+    "area",
```

Comment:
We could redact `address` and `contactInfo` to reduce clutter
```suggestion
TO_REDACT_SITES = {
    "id",
    "siteKey",
    "addresss",
    "contactInfo",
    "siteId",
    "masterBackPlateId",
```